spec/helpers/admin_flash_helper_spec.rb

```diff
@@ -2,18 +2,17 @@
 
 describe AdminFlashHelper do
 
-  describe "#disabled_flash_ids_from_cookies" do
+  describe '#disabled_flash_ids_from_cookies' do
+    let(:cookies) {
+      {
+        'admin_flash_10' => 'disabled',
+        'admin_flash_20' => 'disabled',
+        'not_a_flash_15' => 'disabled',
```

Comment:
Add another key-value pair of `'admin_flash_' => 'disabled'` and make sure that 0 is not included in the returned array